aten/src/ATen/nnapi/nnapi_bind.h

```diff
@@ -36,7 +36,19 @@ struct NnapiCompilation : torch::jit::CustomClassHolder {
     NnapiCompilation() = default;
     ~NnapiCompilation() override = default;
 
-    TORCH_API void init(at::Tensor serialized_model_tensor, std::vector<at::Tensor> parameter_buffers);
+    TORCH_API void init(
```

Comment:
Maybe comment that this is only necessary to support older models that still call this overload.